docs/transactions.md

```diff
@@ -741,6 +741,42 @@ Should be sent to trusted nodes only.
 - REST API:
   - `/dcl/compliance/device-software-compliance`
 
+#### UPDATE_COMPLIANCE_INFO
+
+**Status: Implemented**
+
+Updates a compliance info by VID, PID, Software Version and Certification Type.
+
+
+- Parameters:
+  - vid: `uint16` -  model vendor ID (positive non-zero)
+  - pid: `uint16` -  model product ID (positive non-zero)
+  - softwareVersion: `uint32` - model software version
+  - certificationType: `string` - Certification type - Currently 'zigbee', 'matter', 'access control', 'product security' types are supported
+  - certificationDate: `optional(string)` - The date of model certification (rfc3339 encoded), for example 2019-10-12T07:20:50.52Z
+  - cdCertificateId: `optional(string)` - CD Certificate ID 
+  - reason `optional(string)` - optional comment describing the reason of the certification
+  - cDVersionNumber `optional(uint32)` - optional field describing the CD version number, must be the same with the associated model version
```

Comment:
Please mention that it's `optional(string)`, and that we expect unit32 value here which must be the same as in the associated model version